### PR TITLE
Update ETS2

### PR DESCRIPTION
**Pull request by QuintenQVD0**
# Description

- added 1 extra value to the config as it is needed
- moved to the new steamcmd image

## Checklist for all submissions

* [x] Have you followed the guidelines in our [Contributing document](https://github.com/parkervcp/eggs/blob/master/CONTRIBUTING.md)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../pulls) for the same update/change?
* [x] Have you tested and reviewed your changes with confidence that everything works?
* [x] Did you branch your changes and PR from that branch and not from your master branch?
  * If not, why?:

### CODE REVIEW

**Review by parkervcp — APPROVED** [review]

File: game_eggs/steamcmd_servers/truck-simulator/euro-truck-simulator2/egg-euro-truck-simulator2-dedicated-server.json [hunk 1]
```diff
@@ -99,6 +79,26 @@
             "user_editable": true,
             "rules": "required|string|max:64",
             "field_type": "text"
+        },
```
Comment (gOOvER) [inline game_eggs/steamcmd_servers/truck-simulator/euro-truck-simulator2/egg-euro-truck-simulator2-dedicated-server.json:82]: This is not needed and should be removed

Comment (gOOvER) [reply game_eggs/steamcmd_servers/truck-simulator/euro-truck-simulator2/egg-euro-truck-simulator2-dedicated-server.json:82]: User and password

Comment (QuintenQVD0) [reply game_eggs/steamcmd_servers/truck-simulator/euro-truck-simulator2/egg-euro-truck-simulator2-dedicated-server.json:82]: you mean the query port or the AUTH? as username and password are nuablle

Comment (gOOvER) [reply game_eggs/steamcmd_servers/truck-simulator/euro-truck-simulator2/egg-euro-truck-simulator2-dedicated-server.json:82]: > you mean the query port or the AUTH? as username and password are nuablle

User and password

idk why its used the wrong line :/

**Pull request merged**